src/components/Constants.js

```diff
@@ -0,0 +1,12 @@
+export const MainComponentsSizes = {
```

Comment:
consider adding `lg` :)

Reply:
The reason here is missing `lg` is that I'm considering hide the mobile menu in `lg` mode